src/Playwright/Transport/WebSocketTransport.cs

```diff
@@ -104,13 +104,17 @@ private async Task DispatchIncomingMessagesAsync()
                     if (_webSocket.State != WebSocketState.Open)
                         break;
 
-                    memoryStream.Write(buffer.Array, 0, result.Count);
+#pragma warning disable CA1835 // Change the 'WriteAsync' method call to use the 'Stream.WriteAsync(ReadOnlyMemory<byte>, CancellationToken)' overload
+                    await memoryStream.WriteAsync(buffer.Array, 0, result.Count).ConfigureAwait(false);
+#pragma warning restore CA1835 // Change the 'WriteAsync' method call to use the 'Stream.WriteAsync(ReadOnlyMemory<byte>, CancellationToken)' overload
 
                     if (result.EndOfMessage)
                     {
                         string output = Encoding.UTF8.GetString(memoryStream.ToArray());
                         MessageReceived?.Invoke(this, new MessageReceivedEventArgs(output));
```

Comment:
didn't fix the linter suggestions - since we eventually get rid of the WebSocketTrnasport.cs